File: Python/securityPin.py
import os

def clearConsole():
    command = 'clear'
    if os.name in ('nt', 'dos'):  # If Machine is running on Windows, use cls
        command = 'cls'
    os.system(command)

pin = 0
e = 0
guessedPin = 0

pin = input("Set your PIN: ")

clearConsole()

guessedPin = input("Please enter your PIN: ")

File: C++/securityPin/securityPin.cpp
#include <iostream>
using namespace std;

/* Set a PIN, and then enter that PIN. */

int main(){
    
    int pin, guessedPin, e = 0;

    cout << "Set your PIN: ";
    cin >> pin;
    system("clear");

    do {
        cout << "Please enter your PIN: ";
        cin >> guessedPin;
        if(guessedPin != pin){
            e++;
        }
    } while(e < 3 && guessedPin != pin); 

    if(guessedPin == pin){
        cout << "Accepted\n";
    }
    else{
        cout << "Denied\n";
    }
}